apps/website/src/examples/Searchbox.main.tsx

```diff
@@ -4,11 +4,32 @@
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
 import { SearchBox, Flex } from '@itwin/itwinui-react';
+import { SvgCloseSmall } from '@itwin/itwinui-icons-react';
 
 export default () => {
+  const [inputValue, setInputValue] = React.useState<string>('');
   return (
     <Flex style={{ width: '70%' }}>
-      <SearchBox aria-label='Search input' inputProps={{ placeholder: 'Search...' }} />
+      <SearchBox aria-label='Search input'>
```

Comment:
1. main example should be a bit simpler than this.
2. `aria-label` on here will do nothing because it is applied on the `<div>`

Reply:
Simplified main example.